atomic_reactor/util.py

```diff
@@ -1481,6 +1481,9 @@ class OSBSLogs(object):
     def __init__(self, log, platforms: list):
```

Comment:
if we want to handle None, platforms should be `Optional[list]` (or `Optional[List[str]]`)

Reply:
updated to handle list only